### PR TITLE
Even odd numbers arrenging programme

### PR DESCRIPTION
**Pull request by iamsandipanbera**
Issue #3

### CODE REVIEW

**Comment by iamsandipanbera** [comment]
Please Take a look #6

File: C++/Language/Even Odd Arranging.cpp [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#include<iostream>
```
Comment (khagapati-bagh) [inline C++/Language/Even Odd Arranging.cpp:1]: add a space after #include

File: C++/Language/Even Odd Arranging.cpp [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+            swap(arr[i], arr[j]);
+        }
+        j++;
+    }
+    for (int i = 0; i < n; i++)
+        cout << arr[i] << " ";
+}
+int main()
+{
+    int arr[] = { 2,4,5,1,7,8,9,7};
```
Comment (khagapati-bagh) [inline C++/Language/Even Odd Arranging.cpp:20]: put space in between the numbers like 2, 4, 5

**Review by khagapati-bagh — COMMENTED** [review]
HI @iamsandipanbera  
Thank you for contributing to this repo
Here are some comments for you
1. rename your file as evenOddArrangement.cpp
2. reformat your code, space is missing
#include <iostream>
{2, 4, 5, 1, 7, 8, 9, 7};

**Comment by iamsandipanbera** [comment]
Take a look

**Review by khagapati-bagh — COMMENTED** [review]
add a block for the, for loop at line no 15
add a one-line comment for your program at the top of your file, so people will know what is the program for

**Comment by iamsandipanbera** [comment]
Anything else?

**Comment by khagapati-bagh** [comment]
> add a block for the, for loop at line no 15 add a one-line comment for your program at the top of your file, so people will know what is the program for

@iamsandipanbera you have missed my first point 
add a block for the, for loop at line no 16 like
for (int i = 0; i < n; i++) {
        cout << arr[i] << " ";
}

**Comment by iamsandipanbera** [comment]
Done. Check out Please.

**Pull request merged**